### ISSUE TITLE
Provide precompiled scripts for AEM 6.5.21

### ISSUE THREAD

**Issue by henrykuijpers** [Adobe-Consulting-Services/acs-aem-commons#3403]
## Required Information

* [ ] AEM Version, including Service Packs, Cumulative Fix Packs, etc: 6.5.21
* [ ] ACS AEM Commons Version: 6.6.2
* [ ] Reproducible on Latest? yes

It would be nice to have precompiled scripts for ACS AEM Commons. :) This way, with on-premise AEM, we could leverage faster execution times and no more delays due to waiting for scripts to compile ad-hoc.

**Comment by kwin** [comment]
AFAIK this is only supported in AEMaaCS: https://github.com/adobe/aem-core-wcm-components/issues/1706

**Comment by henrykuijpers** [comment]
It also works in AEM 6.5 SP21, because of all the dependency bumps, including the servlets resolver bump. :) 

I precompiled all the scripts in our projects.
Precompiled all the scripts in AEM.
And now ACS AEM Commons (and wcm.io libraries) are the only ones that have their scripts still go through adhoc compilation.

**Comment by kwin** [comment]
Without having an official confirmation from Adobe side on this, I would be a bit reluctant to do this.

**Comment by henrykuijpers** [comment]
I agree on that. Hopefully this documentation will be there soon. :) 

On the other hand, it shouldn't hurt for AEMaaCS users, as they can already leverage these precompiled scripts.

**Comment by henrykuijpers** [comment]
It works since SP21. That service pack bumps a whole lot of dependencies in AEM. Including the servlets resolver bundle, which contains the precompiled scripts support.

**Comment by kwin** [comment]
Currently the uber-jar for 6.5.21 generates import-package version ranges not resolvable in the newest AEMaaCS (https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/3406#issuecomment-2271817631).
